This adds videos for Seasons 4-6 (#4177)
influxdata/docs-v2: content/resources/videos/API-Invocable-Scripts.md

---
title: API Invocable Scripts
description: >

>   InfluxDB has a feature that lets users invoke scripts using its API. Here, Jay Clifford explains how to use this feature with custom endpoints to improve the functionality, efficiency, and security of your applications.

Reviewer: ```suggestion
  InfluxDB lets you [invoke scripts](/influxdb/cloud/api-guide/api-invokable-scripts/) using the [InfluxDB API](/influxdb/cloud/api/). Here, Jay Clifford explains how to use this feature with custom endpoints to improve the functionality, efficiency, and security of your applications.
```
